lib/l10n/app_th.arb

```diff
@@ -32,6 +37,7 @@
   "detailTabLabel": "Detail",
   "commentTabLabel": "Comment",
   "followupTabLabel": "Followup",
+  "authroityNameLabel": "หน่วยงาน",
```

Comment:
authorityNameLabel

Reply:
fix in https://github.com/onehealthtoolkit/ohtk-mobile/pull/28/commits/7735a2623ff890b05fff1c95727ce74897cfe96b